sphinx_mdolab_theme/ext/embed_n2.py

```diff
@@ -24,13 +24,13 @@ class EmbedN2Directive(Directive):
     Optional arguments are numerical width and height of the embedded object, and
     "toolbar" if the toolbar should be visible by default.
```

Comment:
So the default is to show the toolbar right? Then this docstring should be updated.

Reply:
Good catch. I updated part but not all of it